### DISCUSSION TITLE
关于Doris left join 的问题，求解答

### DISCUSSION THREAD

**Discussion by zzz222z** [apache/doris#11679] [A - General / Q&A]
WITH last AS ( SELECT id FROM customer ORDER BY id DESC LIMIT 10 ),
first_ AS ( SELECT id FROM customer ORDER BY id LIMIT 10 ) 
SELECT
cc.id 
FROM
	customer c
	LEFT JOIN last ON last.id = c.id
	LEFT JOIN first_ ON first_.id = c.id 
WHERE
	last.id IS NOT NULL
这样的sql 为啥查不出来数据，我应该怎么去修改，求解答

**Comment by LemonLiTree** [comment]
cc.id? cc是哪张表？

**Reply by zzz222z** [reply]
写错了 ，cc.id 就是c.id

**Comment by dataalive** [comment]
这个查不出来数据具体是指查出来是空的，还是卡主了跑不完？
